front/components/assistant_builder/types.ts

```diff
@@ -147,3 +141,90 @@ export function getDefaultAssistantState(): AssistantBuilderState {
     },
   };
 }
+
+export function getDefaultRetrievalSearchActionConfiguration() {
+  return {
+    type: "RETRIEVAL_SEARCH",
+    configuration: {
+      dataSourceConfigurations: {},
+      timeFrame: {
+        value: 1,
+        unit: "month",
+      },
+    } as AssistantBuilderRetrievalConfiguration,
```

Comment:
This will not break because it's a litteral !

Reply:
So if the type changes, you'll get a type error here. "as" is useful here so that the caller of this function gets the correct type for `.configuration`  instead of `AssistantBuilderActionConfiguration`